DeepSpeech.py

```diff
@@ -810,7 +757,9 @@ def export():
     saver = tfv1.train.Saver()
 
     # Restore variables from training checkpoint
-    checkpoint = tf.train.get_checkpoint_state(FLAGS.checkpoint_dir)
+    if not FLAGS.save_checkpoint_dir:
+        FLAGS.save_checkpoint_dir = FLAGS.checkpoint_dir
```

Comment:
This should go to the top of the procedure and needs some more elaborate checking (`FLAGS.load_checkpoint_dir`) to better support test-only runs.